README.md

```diff
@@ -376,6 +378,28 @@ BigtableDataSettings.enableOpenCensusStats();
 BigtableDataSettings.enableGfeOpenCensusStats();
 ```
 
+You can view the metrics on the Google Cloud Platform Console
+[Metrics explorer](https://console.cloud.google.com/monitoring/metrics-explorer)
+page.
+
+You can configure how frequent metrics are pushed to StackDriver and the
+[Monitor Resource](https://cloud.google.com/monitoring/api/resources) by
+updating the stats configuration:
+
+``` java
+// set export interval to 10 seconds and monitor resource to global
```

Comment:
nit: Prefix this comment w/ Example, just to make it clear this isn't the standard way to do it.

Reply:
we probably don't need the specified monitored resource too. Can we link to the Java doc for this so people can easily see all the options?